README.md

```diff
@@ -1,85 +1,86 @@
 generation of a set of EPW files for a given set of WMOs and years. It is provided as both a set
-of scripts and as a package that allows EPW files to be generated via function call.
+of scripts (https://github.com/IMMM-SFA/diyepw-scripts) and as a Python package (https://github.com/IMMM-SFA/diyepw).
+This allows DIYEPW to be used as a command-line tool, or as a package to incorporate EPW file 
+generation into a custom script.
 
-## Scripts
-This section describes the scripts available as part of this project.
+# Getting Started
+The DIYEPW Python package can be easily installed using PIP:
 
-Every script has a manual page that can be accessed by passing the "--help" option to the script.
-For example:
- 
 ```
-python analyze_noaa_data.py --help
+pip install diyepw
 ```
 
-## 1. analyze_noaa_data.py
-
-The script analyze_noaa_data.py will assess the files in `outputs/NOAA_AMY` folder for:
-
-1. Total number of rows missing
-1. Maximum number of consecutive rows missing
-
-and will produce the following files (as applicable) under `outputs/analyze_noaa_data_output/`:
-
-1. `missing_total_entries_high.csv`: A list of files where the total number of rows missing exceeds a threshold.
-   The threshold is set to rule out files where more than 700 (out of 8760 total) entries are missing entirely
-   by default, but a custom value can be set with the --max-missing-rows option:
-       
-    ```
-    python analyze_noaa_data.py --max-missing-rows=700
-    ```
-   
-1. `missing_consec_entries_high.csv`: A list of files where the maximum consecutive number of rows missing exceeds 
-   a threshold. The threshold is currently set to a maximum of 48 consecutive empty rows, but a custom value can 
-   be set with the --max-consecutive-missing-rows option:
-   
-   ```
-   python analyze_noaa_data.py --max-consecutive-missing-rows=48
-   ```
-   
-1. `files_to_convert.csv`: A list of the files that are deemed to be usable because they are neither missing too many
-   total nor too many consecutive rows. This file determines which EPWs will be generated by the next script, and
-   it can be freely edited before running that script. Simply remove rows for any files that you do not want the 
-   next script to process.
-
-# 2. Create AMY EPW file
-
-The script create_amy_epw.py will create EPW files by using information for selected variables from the
-NOAA AMY files to replace the values for those same variables in a TMY3 file for the same station number. 
-That is, the following 5 variables will consist of AMY data taken from the NOAA ISD Lite dataset with the remainder
-of the file consisting of the corresponding WMO station's TMY3 EPW file, taken
-from the `inputs/EnergyPlus_TMY3_EPW` directory.
+One you've installed the package, you can access any of the DIYEPW functions or classes by importing the package
+into your own Python scripts:
+
+```
+import diyepw
+diyepw.create_amy_epw_files_for_years_and_wmos([723154, 772104], [2010, 2011, 2012])
```

Comment:
This example doesn't work, since currently 'max_records_to_interpolate', 'max_records_to_impute', and 'max_missing_amy_rows' are required arguments.

Based on the rest of the documentation, I think I'd expect these params to have the default values that are mentioned

Reply:
also, 723154 and 772104 don't seem to exist in the downloadable catalog -- better to choose some other example WMOs

Reply:
Yeah, the WMOs were just made up on the spot, you're right that any example code should work. Updated.